src/client.ts

```diff
@@ -2602,10 +2602,13 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
     );
   }
 
-  async getMessage(messageID: string) {
-    return await this.get<GetMessageAPIResponse<StreamChatGenerics>>(
-      this.baseURL + `/messages/${encodeURIComponent(messageID)}`,
-    );
+  async getMessage(messageID: string, show_deleted_message?: boolean) {
+    let url = this.baseURL + `/messages/${messageID}`;
```

Comment:
Please check the [signature of `StreamChat.get()` method](https://github.com/GetStream/stream-chat-js/blob/a80813f6cc1ec83fcfc770d4196146db3bbcef55/src/client.ts#L1019). It allows you to pass params object so that you do not have to concatenate the URL search params in this way.

Reply:
Oh thanks, fixed it :D